Tools/CISupport/ews-build/steps.py

```diff
@@ -169,30 +169,19 @@ def get_pr_json(self, pr_number, repository_url=None):
             return None
         return pr_json
 
-    def _is_pr_closed(self, pr_number, repository_url=None):
-        if not pr_number:
-            self._addToLog('stdio', 'Skipping pull request status validation since pull request is None.\n')
-            return -1
-
-        pr_json = self.get_pr_json(pr_number, repository_url)
+    def _is_pr_closed(self, pr_json):
         if not pr_json or not pr_json.get('state'):
-            self._addToLog('stdio', 'Unable to fetch pull request {}.\n'.format(pr_number))
+            self._addToLog('stdio', 'Cannot determined closed status, no pull request details provided.\n')
```

Comment:
Nit: 'details provided' would be misleading to the users, since they are not expected to provide any details. Since the immediately prior log would be: "Unable to fetch pull request {}", you can frame this statement something like:
'Cannot determine pull request status.'